### ISSUE TITLE
Clearly explain consensus/execution split in book

### ISSUE THREAD

**Issue by onbjerg** [paradigmxyz/reth#3391]
### Describe the change

We should clearly explain why you need to run a consensus client alongside reth, even if you do not intend to run validators. This can still be a point of confusion for some.

We should probably at least add it here: https://paradigmxyz.github.io/reth/run/mainnet.html#running-reth-on-ethereum-mainnet-or-testnets

A visual to hammer the point home could be good too, like a diagram.

The reason one needs to run a consensus client is:

- Consensus clients decide what blocks are part of the chain
- Execution clients only validate that transactions and blocks are valid in themselves and with respect to world state

In other words: Execution clients execute blocks and transactions, and check that they are valid. Consensus clients decide what valid blocks are part of the chain
